docs/how-to-guides/meson-args.rst

```diff
@@ -44,6 +44,20 @@ building Python wheel. User options specified via ``pyproject.toml``
 or via Python build front-end config settings can be used to override
 the ``meson-python`` defaults..
 
+When building on Windows, ``meson-python`` invokes the ``ninja``
+command via the ``meson compile`` wrapper. When the GCC or the LLVM
+compilers are not found on the ``$PATH``, this activates the Visual
+Studio environment and allows ``ninja`` to use the MSVC compilers. To
+activate the Visual Studio environment unconditionally, pass the
+``--vsenv`` option to ``meson setup``, see this :ref:`example
```

Comment:
We should mention ``--setup-args`` here instead of ``meson setup``.

Reply:
What's `--setup-args`? How to pass arguments to `meson setup` is described two paragraph above and repeated in the linked example. I don't think it is necessary to repeat it.

Reply:
I was referring to the cmdline version of ``[tool.meson-python.args]``

Users don't have direct access to ``meson setup`` through meson-python, so I think mentioning it confusing.

Maybe we can reword the previous sentence as ``pass the --vsenv option as a setup argument`` followed by the link to the example.

Reply:
The diff does not make it obvious, but the page that being amended is title "Passing arguments to Meson". The man topic of the page is how to pass arguments to `meson setup`. It is explained in details two paragraphs above.

Reply:
I don't have a strong opinion on this, so if you really feel like the current approach is fine, I'm OK I guess.

Reply:
What needs to be done is to pass an argument to `meson setup`, which is the consumer of the argument. The command line config setting or the `pyproject.toml` setting are the way you use to specify the argument when using `meson-python`. The way I would word this would be "pass the `--vsenv` option to `meson setup` via the `setup-args` config setting or the `tool.meson-python.meson.args.setup` setting in `pyproject.toml`." But this is just a repetition of what is written ten lines above.